scripts/Fasta2UShER.py

```diff
@@ -65,46 +93,67 @@
 
 """
 
-Retreave problematic sites
+Read in reference
 
 """
-with open(args['reference'],'r') as f:
+with open(args['reference'], 'r') as f:
     for line in f:
         head = line.strip('>').strip()
         break
 
 
+"""
+
+Run fastaToVcf
+
+"""
+
 probDic = {}
-if args['auto_mask'] != None:
+
+# Retrieve recomended problematic sites if the user specied and run fastaToVcf
+
+if args['auto_mask'] is not None:
 
     os.system('rm -r {0}/problematic_sites_sarsCov2.vcf'.format(filePath))
     os.system('wget https://raw.githubusercontent.com/W-L/ProblematicSites_SARS-CoV2/master/problematic_sites_sarsCov2.vcf')
-    if '.vcf' in args['output']: 
-        os.system('{3}/faToVcf -maskSites={3}/problematic_sites_sarsCov2.vcf -ref=\"{0}\" {1} {2}'.format(head,msaName,args['output'].strip()))
+    if '.vcf' in args['output']:
+        os.system(
+            '{3}/faToVcf -maskSites={3}/problematic_sites_sarsCov2.vcf -ref=\"{0}\" {1} {2}'.format(
+                head, msaName, args['output'].strip()))
     else:
-        os.system('{3}/faToVcf -maskSites={3}/problematic_sites_sarsCov2.vcf -ref=\"{0}\" {1} {2}.vcf'.format(head,msaName,args['output'].strip()))
+        os.system(
+            '{3}/faToVcf -maskSites={3}/problematic_sites_sarsCov2.vcf -ref=\"{0}\" {1} {2}.vcf'.format(
+                head, msaName, args['output'].strip()))
 
 
+# Retrieve user provided problematic sites if the user specied and run
+# fastaToVcf
 
-elif args['user_specified_mask'] != None:
+elif args['user_specified_mask'] is not None:
     if '.vcf' in args['output']:
-        os.system('{3}/faToVcf -maskSites={3}/{0} -ref=\"{1}\" {2} {3}'.format(args['user_specified_mask'],head,msaName,args['output'],filePath))
+        os.system('{3}/faToVcf -maskSites={3}/{0} -ref=\"{1}\" {2} {3}'.format(
+            args['user_specified_mask'], head, msaName, args['output'], filePath))
 
     else:
-     
 
-        os.system('{3}/faToVcf -maskSites={3}/{0} -ref=\"{1}\" {2} {3}.vcf'.format(args['user_specified_mask'],head,msaName,args['output'],filePath))
+        os.system('{3}/faToVcf -maskSites={3}/{0} -ref=\"{1}\" {2} {3}.vcf'.format(
+            args['user_specified_mask'], head, msaName, args['output'], filePath))
+
+
+# Else run fastaToVcf without masking
 
 else:
 
     if '.vcf' in args['output']:
-        os.system('{3}/faToVcf  -ref=\"{0}\" {1} {2}'.format(head,msaName,args['output'],filePath))
+        os.system('{3}/faToVcf  -ref=\"{0}\" {1} {2}'.format(head,
```

Comment:
I think Art also wanted that we get rid of os.system calls

Reply:
I thought it was just for rm -r on temp files

Reply:
will replace with subprocess tho